### ISSUE TITLE
Some tests fail on Windows

### ISSUE THREAD

**Issue by k-takata** [Shougo/vimproc.vim#219]
(Copied from #218)
Some tests fail on Windows:

```
not ok 23 - system cmd_system
# The equivalent values were expected, but it was not the case.
#
#     expected: '"\"Foo\""
# '
#          got: ''
```
`vimproc#cmd#system()` doesn't seem to work?

```
not ok 24 - system system_passwd1
# The equivalent values were expected, but it was not the case.
#
#     expected: '-n "test"
# '
#          got: '-n test
# '
not ok 25 - system system_passwd2
# The equivalent values were expected, but it was not the case.
#
#     expected: '-n "test"
# '
#          got: '-n test
# '
```

Maybe above two are caused by the difference of command line parsing.

```
not ok 37 - functions functions
# The true value was expected, but it was not the case.
#
#     expected: true
#          got: 0
```

`vimproc#kill()` doesn't seem to work as expected.

**Comment by Shougo** [comment]
> vimproc#kill() doesn't seem to work as expected.

Hm...   Can you analyse the implementation?

**Comment by Shougo** [comment]
I have fixed some tests.

> vimproc#cmd#system() doesn't seem to work?

I think `vimproc#cmd#system()` is broken.
Can you fix it?

**Comment by Shougo** [comment]
Fixed by #221.
